### ISSUE TITLE
Add support for `NXcanSAS` `@uncertainties`

### ISSUE THREAD

**Issue by t20100** [silx-kit/h5web#1201]
### Is your feature request related to a problem?

The Nexus application definition [NXcanSAS](https://manual.nexusformat.org/classes/applications/NXcanSAS.html) defines an alternative way to provide the error bars in a NXdata through `@uncertainties` attributes

### Requested solution or feature

For curves:
- if there is no `errors` dataset in the `NXdata` group, look for `@uncertainties` attribute of the signal dataset.
- Eventually look for `@uncertainties` attribute on the axes as well to get horizontal error bars (but not needed in current use case).

### Alternatives you've considered

Stick to `NXdata` and do not support `NXcanSAS`.
BTW, when writing a `NXcanSAS` group, one can add an `errors` soft link to the `Idev` dataset to comply to `NXdata`.

### Additional context

A sample file is available here: http://www.silx.org/pub/h5web/test_canSAS.nx

**Comment by t20100** [comment]
BTW, have a look at the discussion in PR https://github.com/silx-kit/silx/pull/3657.
TL;DR: Not necessarily a good idea to support it as it is not the way to inform about `uncertainties` in a `NXdata`.

**Comment by loichuder** [comment]
Generally, I don't think it is in our interest to start supporting application definitions features. 

Given their number, I fear we might open the Pandora's box.

**Comment by woutdenolf** [comment]
We don't need this. You can save data which is compliant with NXdata and NXcansas at the same time.
